js/everything.js

```diff
@@ -474,7 +473,13 @@ $(function() {
             );
         }
     }
-
+    if ( /Android|webOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini/i.test(navigator.userAgent) )
+    {
+        $("#container").ready(function() {
```

Comment:
@shakeelmohamed Added the listener as a formality , will remove it and make the changes mentioned.